.github/workflows/ci.yaml

```diff
@@ -0,0 +1,43 @@
+name: CI
```

Comment:
The scripts for these have not been commited yet. Can you add them to PR, Else the workflow will fail

Reply:
Added in commit [0c418b6](https://github.com/opensearch-project/opensearch-api-specification/pull/35/commits/0c418b61d7d19740631a77e74127b18f653f312c).